routes/api.php

```diff
@@ -14,6 +15,30 @@
 |
 */
 
+// Get all todo routes
+Route::get('todos', [TodoController::class, 'index']);
```

Comment:
Please use resource routes and keep REST API principles. 

https://laravel.com/docs/10.x/controllers#resource-controllers